komodoenv/creator.py

```diff
@@ -22,7 +22,15 @@ def open_chmod(path: Path, mode: str = "w", file_mode=0o644):
 class Creator:
     _fmt_action = "  " + green("{action:>10s}") + "    {message}"
 
-    def __init__(self, komodo_root, srcpath, trackpath, dstpath=None, use_color=False):
+    def __init__(
+        self,
+        komodo_root,
+        srcpath,
+        trackpath,
+        dstpath=None,
+        *,
+        use_color=False,
+    ):
```

Comment:
Can you clarify why the `*` has been placed exactly here?
If we want to enforce positional arguments, why not one arg earlier?